tests/test_parser.py

```diff
@@ -0,0 +1,76 @@
+import os
+import sys
+
+THIS_FOLDER = os.path.dirname(os.path.abspath(__file__)) + "/.."
```

Comment:
this is very janky - what are you trying to achieve? It seems like you're not running your code correctly if you need this

Reply:
Doesn't seem that bad, but maybe we could use relative imports (`...`) instead?

Reply:
seems like something necessary to keep the imports clean. @Lyxal thoughts?